streams/src/test/java/org/apache/kafka/test/NoOpReadOnlyStore.java

```diff
@@ -78,6 +78,7 @@ public void init(final ProcessorContext context, final StateStore root) {
             new File(context.stateDir() + File.separator + name).mkdir();
         }
         this.initialized = true;
+        context.register(root, (k, v) -> { });
```

Comment:
Why do we need this?

Reply:
That is what the real code does, too. Otherwise we need to call `register` manually in the tests what is annoying.